ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -1090,6 +1172,26 @@ Long bootStrapDump(Path dumpRoot, DumpMetaData dmd, Path cmRoot, Hive hiveDb)
     }
   }
 
+  private void unsetReplFailoverEnabled(Hive hiveDb, Database db) throws HiveException {
```

Comment:
Just  unsetReplFailoverEnabled(Hive hiveDb) is enough no?

Reply:
How will this fxn signature know for which db it has to unset this prop?